jest-preset.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
       },
     },
   },
+  testEnvironment: 'jest-environment-jsdom-sixteen',
```

Comment:
isn't latest Jest using jsdom v16?

Reply:
for some reasons, my latest push isn't affected

Reply:
should be good now :)